code-for-canada/project-thundercat
# [Look and Feel] Side Navigation in Test

# Description

Creation of generic Side Navigation component; update of CSS to have proper look and feel. (Definition specific to eMiB was in a previous PR).

## Type of change

Please delete options that are not relevant.

- New feature (non-breaking change which adds functionality)
- Code cleanliness or refactor

## Screenshot

Please provide if applicable (browser for `frontend` or any visual way to show a `backend` change).

![image](https://user-images.githubusercontent.com/2746350/53358418-46ab7080-38fe-11e9-8725-aa78a49f7389.png)

![image](https://user-images.githubusercontent.com/2746350/53358404-372c2780-38fe-11e9-98c1-1d38b89918be.png)

![image](https://user-images.githubusercontent.com/2746350/53358433-52973280-38fe-11e9-8bc1-ac5bf5e76787.png)

![image](https://user-images.githubusercontent.com/2746350/53358446-5b880400-38fe-11e9-9bfa-6e2ce5f96605.png)

# Testing

Applicable for all code changes.

Manual steps to reproduce this functionality:

1.  Start eMIB
2.  Navigate through the pre-instruction pages
3.  Open the Instructions Tab
4.  Verify that the look and feel is correct and functions as expected

NOTE:
- Test tabs and side navigation are not locked to the top of the page
- The other instruction components/pages will not show up until you click the button
- Spacing and coloring for the TestTabs component will be a later PR (that will also fix the coloration).

Screenshot of unit tests run passing:
![image](https://user-images.githubusercontent.com/2746350/53358460-69d62000-38fe-11e9-8d1a-59dc764aca73.png)


# Checklist

Applicable for all code changes.
